src/main/scala/dsptools/Chisel3TestersCompatibility.scala

```diff
@@ -0,0 +1,14 @@
+package chisel3.iotesters
```

Comment:
@ducky64  @shunshou I think we should consider looking into what you need here and add it, if at all possible, to chisel3 API.  We are doing a lot of API cleanup right now, so it's a good time.  I see only one use of it, can you describe the use case